api/server/httpstatus/status.go

```diff
@@ -67,10 +66,11 @@ func FromError(err error) int {
 			return FromError(e.Cause())
 		}
 
-		log.G(context.TODO()).WithFields(logrus.Fields{
+		log.G(context.TODO()).WithFields(log.Fields{
 			"module":     "api",
+			"error":      err,
 			"error_type": fmt.Sprintf("%T", err),
-		}).Debugf("FIXME: Got an API for which error does not match any expected type!!!: %+v", err)
+		}).Debug("FIXME: Got an API for which error does not match any expected type!!!")
```

Comment:
Thought I'd also add the error as structured field here, as we were already were passing fields.